vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthVariant.kt

```diff
@@ -110,7 +111,7 @@ class FtueAuthVariant(
     }
 
     override fun setIsLoading(isLoading: Boolean) {
-        // do nothing
+        doNothing()
```

Comment:
I would prefer a comment, or write `override fun setIsLoading(isLoading: Boolean) = Unit`. I do not know what others think.

Reply:
Fair enough. This is purely my personal preference, but `Unit` definitely works too